src/emcpy/cfg/var_defaults.yaml

```diff
@@ -88,3 +88,33 @@ surface pressure:
   short name: ps
   units: Pa
   variable type: conventional
+sea surface temperature:
+  cmap: rainbow
+  contour interval: 5
+  diff contour interval: 0.5
+  diff range: [-5, 5]
+  name: sea surface temperature
+  range: [255, 305]
+  short name: sst
+  units: K
+  variable type: conventional
+tc pressure:
+  cmap: rainbow
+  contour interval: 4
+  diff contour interval: 0.5
+  diff range: [-5, 5]
+  name: tc pressure
+  range: [800, 1050]
```

Comment:
in this YAML, surface pressure is in Pa, can we use Pa here too and just make sure to convert when plotting tcp?